Header

AnnaVolkodav opened the pull request.

AnnaVolkodav committed:
структура хедера

AnnaVolkodav committed:
Merge branch 'main' into header

AnnaVolkodav committed:
структура та стилі меню

без іконки і стилів для кнопки-хрестика

AnnaVolkodav committed:
Merge branch 'main' into header

AnnaVolkodav committed:
меню + адаптив

AnnaVolkodav committed:
fixing the size for phone-wrapper in 768 screen

AnnaVolkodav committed:
Merge branch 'main' into header

AnnaVolkodav committed:
fixing menu-nav-list

AnnaVolkodav committed:
Merge branch 'main' into header

AnnaVolkodav committed:
Correcting

Delete the padding-bottom for header-section
Correct the size for header-soc-item
Add the height for header-section
Delete the max-width for .container (mobile, tablet, laptop 1280, laptop 1440)

AnnaVolkodav committed:
Merge branch 'main' into header

AnnaVolkodav committed:
Correct the .menu-wrapper gap and transform: none; for 1280px and

AnnaVolkodav committed:
Update header.html

AnnaVolkodav committed:
added js code for menu

stelmakh1991 commented on src/partials/header.html:
```diff
@@ -2,7 +2,7 @@
   <div class="header-container container">
     <img src="./img/header/logo.svg" alt="logo" width="44" height="39" class="logo-img" />
```
добав srcset з 2x img

AnnaVolkodav committed:
Corrected the value of min-resolution and margin

Corrected the value of min-resolution and changed the margin value for .logo-img (delete for screen 375px and add for screen 1280px)
